iXBRLViewerPlugin/viewer/src/js/tableExport.js

```diff
@@ -243,10 +252,14 @@ export class TableExport {
         }
 
 
-        const wb = this._writeTable(data);
-        wb.xlsx.writeBuffer().then( data => {
-          const blob = new Blob( [data], {type: "application/octet-stream"} );
-          FileSaver.saveAs( blob, 'table.xlsx');
+        const excelData = this._writeTable(data);
+        writeXlsxFile(excelData.data, {
+            columns: excelData.columns,
+            fileName: 'table.xlsx'
         });
+        //wb.xlsx.writeBuffer().then( data => {
+        //  const blob = new Blob( [data], {type: "application/octet-stream"} );
+        //  FileSaver.saveAs( blob, 'table.xlsx');
+        //});
```

Comment:
Remove commented out?

Reply:
Done in [36f1ffd](https://github.com/Workiva/ixbrl-viewer/commit/36f1ffd15a9ed59700864763cd7d3878067d0c2a)